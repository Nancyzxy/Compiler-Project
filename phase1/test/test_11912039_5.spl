int main()
{
    char c[10.0];
    b = c || 2
    mo=m;
    no=n;
    while(m/n>0)
    {
        r=m/n;
        m=n;
        n=r;
    }
}

