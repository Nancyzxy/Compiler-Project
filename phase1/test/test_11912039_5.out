Error type B at Line 3: Wrong type of index
Error type B at Line 4: Missing semicolon ';'
